src/System.Linq/samples/project.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10",
```

Comment:
I've been thinking about this a bit more and I think we shouldn't encounter any issues, as long as we stick to our current workflow, which is keeping all our lock files in sync (usually we just refresh them all at once). It would still be interesting to know what happens regarding project references and different package versions, though, just out of curiosity.